addons/mail/static/src/components/notification_list/tests/notification_list_notification_group_tests.js

```diff
@@ -1,8 +1,9 @@
 /** @odoo-module **/
 
-import { afterEach, afterNextRender, beforeEach, start, } from '@mail/utils/test_utils';
+import { afterEach, beforeEach, start } from '@mail/utils/test_utils';
 
 import Bus from 'web.Bus';
+import {afterNextRender} from "../../../utils/test_utils";
```

Comment:
Not sure what's the goal here, this diff should be reverted.

Reply:
If you remove the existing one from imports, you'll have to get it from start in all other tests in this file. I would suggest no diff in imports on this PR. Just keep the import when it was already there, and take it from start in new tests.